target/fishGDDFig-desktop.svg

```diff
@@ -214,15 +214,12 @@
 			mainDocument.firefoxSVGReload('fishGDDFig');
         	sweepDown(Number(initialHeight)+25); // bump plus a little extra
 			
-			var i = 0;
-			var delaySendValue = setInterval(function () {   
-					    	if (i < dots.length){
-					    		window.parent.ga("send", "event", "user feedback", "post", "GDD guess "+i, getPtValue(i));
-								i++
-							} else {
-			                   clearInterval(delaySendValue);
-							}}, 2000);
-			   
+			var toSend = getPtValue(0);
+			for (var i = 1; i < dots.length; i++) {
+				toSend = toSend + "," + getPtValue(i);
+			}
+			toSend = "{'walleye':["+toSend+"]}"
```

Comment:
Do you have access to JSON from there? It is a non-scoped function, so should just work.